packages/esm-form-entry-app/src/app/app.module.ts

```diff
@@ -18,6 +18,7 @@ import { ConfigResourceService } from './services/config-resource.service';
 import { LoaderComponent } from './loader/loader.component';
 import { SingleSpaPropsService } from './single-spa-props/single-spa-props.service';
 import { FormCreationService } from './form-creation/form-creation.service';
+import * as ngTranslate from '@ngx-translate/core';
```

Comment:
Why are we doing this rather than `import { TranslateService } from '@ngx-translate/core'`?

Reply:
In general, `*` imports should be avoided where at all possible.